packages/next/src/server/app-render/action-handler.ts

```diff
@@ -903,6 +930,7 @@ export async function handleAction({
           skipFlight:
             !staticGenerationStore.pathWasRevalidated || actionWasForwarded,
         }),
+        temporaryReferences: undefined,
```

Comment:
am I missing where you actually return the references? seems it is undefined everywhere